source/core_pki_utils.c

```diff
@@ -170,18 +170,18 @@ int8_t PKI_pkcs11SignatureTombedTLSSignature( uint8_t * pucSig,
             pucSig[ 3 ] = 0x21;                          /* Increment the length of the R value to account for the 0x00 pad. */
             pucSig[ 4 ] = 0x0;                           /* Write the 0x00 pad. */
             ( void ) memcpy( &pucSig[ 5 ], ucTemp, 32 ); /* Copy the 32-byte R value. */
-            pucSigPtr = pucSig + 33;                     /* Increment the pointer to compensate for padded R length.    */
+            pucSigPtr = &pucSig[ 33 ];                   /* Increment the pointer to compensate for padded R length.    */
```

Comment:
```suggestion
            pucSigPtr = &(pucSig[ 33 ]);                   /* Increment the pointer to compensate for padded R length.    */
```

Reply:
Array subscripting operator `[]` seems to have higher precedence than address-of operator `&`.
The extra parentheses do not seem to be necessary.
Reference [this link](https://en.wikipedia.org/wiki/Operators_in_C_and_C%2B%2B#Operator_precedence).